src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,102 @@
+package main.java.duke;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+
+/**
+ * The self-named main class that controls our lovely chatbot, Duke
+ *
+ * @author eugeneleong
+ * @version 1.0
+ */
+
+public class Duke {
+
+    private Storage storage;
+    private TaskList tasks;
+    private Ui ui;
+
+    /**
+     * Loads Duke up with the appropriate list of tasks
```

Comment:
Perhaps a full stop could be added at the end of each line as per coding standard.